pkg/agent/baker_test.go

```diff
@@ -135,6 +135,7 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 			PrimaryScaleSetName:           "aks-agent2-36873793-vmss",
 			IsARM64:                       false,
 			DisableUnattendedUpgrades:     false,
+			SSHStatus:                     datamodel.SSHUnspecified,
```

Comment:
can you add/modify a unit test case for enable/disable ssh and ensure the generated code looks fine? specifically cse_main